### PR TITLE
Chore/lw 10534 translation package

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA -[LW-10534](https://input-output.atlassian.net/browse/LW-10534)
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Move all the translations files and i18n instance into the separate package.

## Testing

All the translations should work the same as before.

## Screenshots

~~Attach screenshots here if implementation involves some UI changes~~

[LW-10534]: https://input-output.atlassian.net/browse/LW-10534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tommayeliog — APPROVED** [review]
LGTM!

**Comment by mchappell** [comment]
@vetalcore `packages/e2e-tests/src/utils/translationService.ts` will need updating to point at the correct `browser-extension-wallet` and `core` translation files, this should get the e2e tests running

**Review by mchappell — CHANGES_REQUESTED** [review]
Once storybook changes have been corrected, happy to pass this on, looks like that's the only remaining change left. Great work 🎉

**Review by mchappell — APPROVED** [review]
Great job on the refactor, much easier to have all this in one place. Did you ever consider moving all translations into a single file?

**Comment by DominikGuzei** [comment]
Yeah i also wonder why the translations are still split up 🤔 since they are already namespaced it would make sense to merge them? (although we would have to namespace some of the extension translations i think)

File: apps/browser-extension-wallet/src/views/browser-view/features/settings/components/TermsDrawer.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import cn from 'classnames';
 import { Drawer, DrawerHeader, DrawerNavigation } from '@lace/common';
-import { LegalTranslations, useTranslate } from '@lace/core';
+import { LegalTranslations } from '@lace/core';
 import styles from './SettingsLayout.module.scss';
+import { useTranslation } from 'react-i18next';
```
Comment (DominikGuzei) [inline apps/browser-extension-wallet/src/views/browser-view/features/settings/components/TermsDrawer.tsx:6]: im not sure if that's the correct import?

**Comment by vetalcore** [comment]
> Yeah i also wonder why the translations are still split up 🤔 since they are already namespaced it would make sense to merge them? (although we would have to namespace some of the extension translations i think)

it's just easier to handle the changes this way, we could easily handle that as a next iteration.

**Review by wklos-iohk — APPROVED** [review]

**Pull request merged**